wsdl/ver10/advancedsecurity/wsdl/advancedsecurity.wsdl

```diff
@@ -2049,6 +2293,64 @@
 					</xs:sequence>
 				</xs:complexType>
 			</xs:element>
+			<!--===============================-->
+			<xs:element name="GetAuthorizationServerConfigurations">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="Token" type="tt:ReferenceToken" minOccurs="0"/>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="GetAuthorizationServerConfigurationsResponse">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="Configurations" type="tas:AuthorizationServerConfiguration" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
is Token should part of AuthorizationServerConfiguration?

Reply:
There's two types AuthorizationServerConfiguration and AuthorizationServerConfigurationData. AuthorizationServerConfiguration  contains the things that can be modified. The token is part of AuthorizationServerConfiguration and can't be modified, it's generated by the device.